src/components/Leaderboard/RescueAchievementIcon.js

```diff
@@ -42,7 +45,7 @@ function RescueAchievementIcon ({ rescueCount, ...iconProps }) {
 
   return Icon && (
     <div
-      className={['achievement rescue-count', { crown: count >= 500 }]}
+      className={classes}
```

Comment:
This can still be written similarly to before like this:

```suggestion
      className={[`${styles.achievement} ${styles.rescueCount}`, { [styles.crown]: count >=500 }]}
```